src/woo_publications/api/urls.py

```diff
@@ -4,10 +4,14 @@
 from drf_spectacular.views import SpectacularJSONAPIView, SpectacularRedocView
 from rest_framework import routers
 
+from woo_publications.metadata.api.viewsets import InformationCategoryViewset
```

Comment:
```suggestion
from woo_publications.metadata.api.viewsets import InformationCategoryViewSet
```
I pointed this out in an earlier review - let's keep the casing consistent with the base classes